temporian/core/data/duration.py

```diff
@@ -23,6 +23,26 @@
 def milliseconds(value: Union[int, float]) -> Duration:
     """Converts input value from milliseconds to a `Duration` in seconds.
 
+    Example:

```

Comment:
Could you add two extra pieces of information:
- In Temporian, timestamps and durations are float64. The user chose the semantic of this value.
- In Temporian, datetime related functions (such as duration, calendar and plotting functions) assume that durations are expressed in seconds (https://temporian.readthedocs.io/en/latest/user_guide/#time-units). So it is a good idea for user to use second unit when it make sense.

Reply:
adding to duration docs